tests/test_speed.py

```diff
@@ -41,11 +47,69 @@ class SpeedTestCase(asynctest.TestCase):
     makes sure they are run automatically and avoids bit rot.
     """
 
+    @classmethod
+    def setUpClass(cls):
+        if verify is None:
+            return
+        metrics = (
+            verify.Metric(
+                name="salobj.CreateClasses",
+                description="The number of topic classes that dds can create "
+                "(from an IDL file), per second. "
+                "This is measured by creating one of of each Test topic. "
```

Comment:
double "of" 

> This is measured by creating one of ~of~ each Test topic.

Reply:
Dang. I missed this comment before I merged. I'll fix it now and it'll get into the next version.